src/engine/specification.md

```diff
@@ -470,15 +470,24 @@ This method follows the same specification as [`engine_forkchoiceUpdatedV1`](#en
 * method: `engine_getPayloadV2`
 * params:
   1. `payloadId`: `DATA`, 8 Bytes - Identifier of the payload build process
+* timeout: 1s
 
 #### Response
 
-* result: [`ExecutionPayloadV2`](#ExecutionPayloadV2)
+* result: `object`
+  - `executionPayloadV2`: [`ExecutionPayloadV2`](#ExecutionPayloadV2)
+  - `blockValue` : `QUANTITY`, 256 Bits - The expected value to be recieved by the `feeRecipient` in wei
 * error: code and message set in case an exception happens while getting the payload.
 
 #### Specification
 
-Refer to the specification for [`engine_getPayloadV1`](#engine_getpayloadv1).
+1. Given the `payloadId` client software **MUST** return the most recent version of the payload that is available in the corresponding build process at the time of receiving the call.
+
+2. The call **MUST** return `-38001: Unknown payload` error if the build process identified by the `payloadId` does not exist.
+
+3. Client software **MAY** stop the corresponding build process after serving this call.
+
+4. Client software **MAY** use the sum of the block's priority fees to determine `blockValue`.
```

Comment:
```suggestion
4. Client software **MAY** use the sum of the block's priority fees or any other algorithm to determine `blockValue`.
```

This addition makes it clearer that there is no algorithm mandated by the Engine API spec, and that this computation may be implementation dependent.